config/routes.rb

```diff
@@ -2755,9 +2755,9 @@
   API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
-  match '/api/*path' => 'manage_i_q/a_p_i/base#handle_options_request', :via => [:options]
+  match '/api/*path' => 'a_p_i/base#handle_options_request', :via => [:options]
 
-  get '/api(/:version)' => 'manage_i_q/a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
+  get '/api(/:version)' => 'a_p_i/base#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
```

Comment:
Had similar issues, even with :controller => "API" specified. One way to having rails support the upper case controller name 'API' can be done with inflectors:

adding a new file config/initializers/inflectors.rb:

```
ActiveSupport::Inflector.inflections do |inflect|
  inflect.acronym 'API'
end
```

worked like a champ.